### DISCUSSION TITLE
遲到不是故意

### DISCUSSION THREAD

**Discussion by delia07152** [1101-GS4719-C/Python-Community-GS4719-C#931] [General]
https://technews.tw/2022/12/12/why-some-people-always-late/

這篇文章為性格導致的遲到族群申辯，啊 遲到的理由又多一項了!
但不論是基於性格還是文化等等各種原因，遲到對於身邊的人來說其實還蠻困擾的，雖然文中提及若是性格所致，可能很難改變，但知道自己一定會遲到，不就應該要加快速度了嗎~
如果朋友遲到，並且提出的理由是因為性格，我想大家都會覺得無言吧!

**Comment by ruthchen1020** [comment]
反正遲到不要影響其他人就好，性格沒有好壞之分。

**Comment by peggy0329** [comment]
我有一個愛遲到的朋友，他真的日子過得超悠閒，感覺他的時區比別人慢1小時？
遲到1個小時真的對別人影響很大，雖然可能跟性格有關
但我認為與他人約定的承諾，不可以用性格來撇清責任吧！

**Comment by Evan0979** [comment]
超討厭遲到的人欸,感覺超不尊重得很沒禮貌!

**Comment by CPCpyt** [comment]
我覺得遲到不要影響他人就好，
畢竟社會上沒人有時間聽你的理由，
只在乎你對他人的利益有否造成損害。

**Comment by StevenWanglolz** [comment]
看來我以前過得太悠哉了才會常常遲到，現在倒是完全相反 :(

**Comment by orianna901001** [comment]
個人不是很喜歡遲到的人，除非對方是很好的朋友，也知道彼此都會遲到，不然遲到真的不是一個好的美德，別人也會為你貼上愛遲到的標籤，我感覺這樣以後在做比較重要的事情時，會發生報應的QQ

**Comment by Jamesmoneya** [comment]
遲到個人覺得還好，只是跟大家約的時候， 會遲到的就是會遲到，所以大家都知道不管我們約幾點我們都會至少晚半個小時到，因為遲到的那個人總是半小時以上，他也不知道我們都晚到，所以他不會因為我們都晚半小時，她就變得更晚半小時，嘿嘿

**Comment by ziihui** [comment]
無法忍受不準時的人😵‍💫 尤其是開會、討論報告的時候，不尊重別人時間的人真的好可惡😡

**Comment by Hahahahbla** [comment]
性格導致的遲到確實是一個複雜的問題，雖然難以完全改變，但通過自我認識和時間管理等方法，可以逐步改善。對於朋友的遲到，理解和包容是必要的，但也需要適時地溝通和提醒，幫助他們認識到問題並進行改進。這樣不僅能減少遲到帶來的不便，也能促進更好的相處和合作。

**Comment by willychiuchiu** [comment]
自己是不喜歡遲到的人，都會抓時間剛剛好到，通常都不會遲到。除非和特定的遲到慣犯約時，在已經確定對方會遲到至少20分鐘的前提下，我就會把約定時間提早20分鐘，然後自己再按照原定的時間抵達，這是我目前想到對付愛遲到人的最佳方法。

**Comment by OG-Matcha** [comment]
我自己覺得準時是必要的尊重，如果和別人約好的話，能早到就不要遲到，講現實點沒有人有義務等你，既然選擇遲到那就要對自己的選擇負起責任、承擔後果。

**Comment by happy246810** [comment]
遲到的原因百百種，主要是看遲到的原因跟遲到多久，如果是因為生理因素(上廁所)之類的問題或遲到幾分鐘而已的話，我個人是可以接受，但那種故意遲到20-30分鐘的人，真的非常討厭。那些因為睡過頭而遲到幾分鐘的人，我也是能接受的，因為你畢竟不知道她前一天幾點睡，在做些什麼，說不定他在再跟他的微積分奮戰對吧。

**Comment by NcuJerryKao** [comment]
這篇文章探討了性格導致的遲到行為及其對他人的影響，以下是幾個建設性建議，幫助提升文章的結構、內容和清晰度：
1. 引言部分的強化：
- 引言可以更具信息性和吸引力，簡要介紹文章的主題和背景，讓讀者明白接下來討論的重點。
- 例如：「遲到是一種常見的現象，但你是否曾經想過，遲到可能與一個人的性格有關？本文將探討性格導致的遲到行為及其對他人的影響。」
2. 段落結構：
- 將文章分成更易於理解的段落，每個段落集中討論一個具體的觀點。例如：
1. 引言：介紹文章主題和背景。
2. 性格導致的遲到：詳細說明性格如何影響遲到行為。
3. 遲到的影響：討論遲到對他人和社會的影響。
4. 解決方法：提供一些實際的解決方法和建議。
5. 結論：總結文章並提出最終的建議。
3. 過渡和連接詞：
- 使用過渡句和連接詞來連接不同段落，使文章流暢。例如，「接下來，我們來看看性格如何影響遲到行為。」或「然而，遲到對他人和社會的影響不容忽視。」
4. 詳細說明和分析：
- 在描述性格導致的遲到行為時，可以更詳細地解釋其背後的心理機制和具體表現。例如，列出哪些性格特質容易導致遲到。
- 例如：「研究表明，某些性格特質如完美主義、注意力分散和時間管理能力不足，可能會導致人們經常遲到。」
5. 語言簡化和澄清：
- 某些段落中的語言較為簡單，可以考慮稍微豐富和澄清。例如，「知道自己一定會遲到，不就應該要加快速度了嗎」可以改為「如果知道自己經常遲到，是否應該採取一些措施來改善時間管理？」
6. 引用和支持：
- 若能引用一些具體的研究或專家觀點來支持論點，將大大增強文章的權威性。例如，可以引用心理學研究或專家觀點來支持「性格導致遲到」的討論。
7. 個人經驗和反思：
- 個人經驗和反思是文章的一大亮點，可以更詳細地描述實際經歷和感受，這樣可以讓讀者更容易產生共鳴。
- 例如，「我有一位朋友經常遲到，並且總是以性格為理由。這讓我們其他人非常困擾，因為我們不得不改變計劃來適應他的遲到。」
8. 結論和建議強化：
- 在結論部分，再次強調文章的主要觀點，並總結文章的重要結論。這有助於讀者記住文章的核心思想。
- 例如：「總結來說，雖然性格可能導致遲到行為，但這並不是一個不可改變的藉口。我們應該努力改進自己的時間管理能力，尊重他人的時間。」
9. 讀者互動：
- 在文章結尾，可以提出一些問題或邀請讀者分享他們的觀點和經驗，以增加互動性和參與感。
- 例如，「你是否有過因為遲到而受到困擾的經驗？你認為應該如何改善時間管理？歡迎在評論區分享你的意見。」
通過這些建議，文章將變得更加結構化、清晰且具吸引力，讀者也會更容易理解和產生共鳴。

**Comment by jylin34** [comment]
我覺得準時是一種尊重，如果是重要的事情然後遲到，完全不能接受，以後可能也不會跟這個人往來，除非是很熟的朋友，約出來玩一玩，遲到個十幾分鐘這種都還可以接受。

**Comment by titan92601** [comment]
我認為遲到雖有性格或文化原因，但並非合理藉口。遲到對他人困擾大，應強化時間管理，如設提醒、提前準備。遲到時應主動通知並道歉，表達對他人時間的尊重。理解遲到原因有助於改進，但責任感和尊重他人時間是避免遲到的關鍵。

**Comment by LinChloe** [comment]
我認為一個人會遲到是他認為此事沒那麼重大。
跟別人有約會遲到。
上課會遲到。
小考敢遲到嗎?
期中期末考敢遲到嗎?
搭公車敢遲到嗎?
搭飛機出國時敢遲到嗎?
都只是看事情的嚴重性而已。
所以除非特殊原由，遲到習慣都只是藉口罷了。

**Comment by 20050201Shawn** [comment]
身為一個以前(現在有好一點)蠻會遲到的人，我可以分享一下我的心態，可能有點自我抱歉。如果是我覺得沒那麼想上的爛課或是我自己覺得不是很重要的聚會，我會想要以自己舒服為優先，但如果是我覺得很重要的面談我就會逼自己準時(但不是提早到，或是只提早到5分鐘)。可能因為以前比較多被強制參加或是沒得選擇老師就會比較想擺爛(還有自制力低)，上大學後才比較多自己的選擇，比較有動機為自己選擇負責，所以會比較主動一點。當然，我覺得有人討厭遲到是理所當然的，還是要尊重別人的感受或事後補償。我覺得我會遲到除了一小部分是習慣導致外，主要還是我覺得重不重要所導致。

**Comment by alexender0104** [comment]
我自己事非常討厭會遲到的人，我覺得既然都事先約好了，那準時到或是提前到，不是對對方的尊重嗎?所以我時常想不懂遲到的人心中想的是甚麼，有些 人可能覺得，遲到一下不會怎樣啦!但你卻是浪費了等待的人的時間，所以我認為習慣性的遲到並不能用不是故意的這種藉口搪塞過去!

**Comment by Shang621** [comment]
遲到有時並非故意，往往是因為突發情況或不可控因素所致。例如，交通堵塞、公共交通延誤、突如其來的家庭緊急情況，甚至是個人健康問題，都可能導致遲到。每個人都有可能面臨這樣的情況，因此我們應該互相理解和包容。當有人遲到時，與其立即指責，不如先聽聽對方的解釋。這樣不僅能夠減少誤會，還能增進彼此的信任和關係。在生活中，寬容和理解是保持良好人際關係的重要因素。

**Comment by hbh10601** [comment]
我個人是看情況，如果是跟人約好要到哪個地點，除非有臨時出狀況，否則我都會提前到
又如果是上課，會看那堂課的重要程度，如果比較Free，就不一定會準時，不過家凱老師的Python我是沒遲到過><。

**Comment by poiu6fi** [comment]
個人覺得如果是平常沒事約出去玩，那種遲到還好。不過如果是開會或是討論一些重要的東西那就不太能接受了

**Comment by 710388** [comment]
遲到就是故意，只能說不遲到的方法很多，但就是有人選擇遲到那絕對是故意的，沒心

**Comment by asd99263fg** [comment]
現在真的對遲到的容忍超大，之前我們要出去玩等一個傢伙等了三個小時，他來的時候還有臉跟我們嘻嘻哈哈，我們都準時然後為了趕時間想花錢去搭高鐵，他還覺得自己沒有錯覺得應該配合他，我真的超無奈

**Comment by asd99263fg** [comment]
其實遲到很多時候都是時間安排的不好導致，所以好好決定自己時間規劃真的很重要，而且從很多小事都可以讓人知道這個人的做人態度還有處事，尤其是準時這個部分，我自己是覺得能堅持把準時做好的人，其他事自然也可以做的很不錯啦

**Comment by cassy0201** [comment]
我覺得知道當天自己會遲到還沒有提前說的人好像更過分了，在現場乾等還不能確定對方是否會出現，真的極度浪費時間

**Comment by tshsu04224** [comment]
我個人就是一個喜歡拖拖拉拉，偶爾可能會遲到的那種，但我因為動作慢，培養出了另一個技能，就是走路超快，既然動作太慢會來不及赴約，那就只好縮短移動時間了，只是副作用是每次跟朋友走路走一走，都會不小心自己越走越前面😓

**Comment by 155Julia** [comment]
看完這篇報導，我覺得雖然科學家提出了性格會導致遲到的解釋，但這並不能完全合理化遲到的行為。遲到不僅會對他人造成困擾，也顯示出對他人時間的不尊重。我認為，即使性格使然，遲到的人仍應該努力改善自己的時間管理，至少要表示歉意並加快赴約速度。若朋友遲到且理由是性格所致，我想大家都會感到無言，我也會感到非常無奈。时间管理是尊重他人的基本礼貌。

**Comment by YeeDaniel** [comment]
遲到喔，這種事情就大學生日常欸，但是認真說啦，可以的話我還是會準時抵達，畢竟尊重是必須要給到的，不然讓別人乾等自己很久，對自己的觀感自然也不會太好。所以還是建議準時抵達，而不是當一個遲到仔

**Comment by asd99263fg** [comment]
為了避免遲到，可以提前做好時間管理，設定鬧鐘並給自己留出充裕的準備時間，提前規劃好交通路線和考慮可能的延誤因素，並養成提前出發的習慣。同時，保持健康的作息，確保充足的睡眠，讓自己每天都有充沛的精力迎接新的挑戰。
